dicomImport/matRad_importDicom.m

```diff
@@ -33,6 +36,12 @@
 %
 % %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+%%
+if ~exist('dicomBool','var')
+  dicomBool = true;
+end
```

Comment:
I think it's a good idea to have this activated by default. Maybe also make this the default stting in the GUI? I think pseudonymization should not be the default option!